### ISSUE TITLE
Invalid ammobox id provided (2)

### ISSUE THREAD

**Issue by xRiseless** [tmp64/weaponmod#1]
When i try to use WeaponMod Spawn Config Editor - menu just disappear and i got this error:
 
```
L 03/21/2021 - 17:35:37: [WEAPONMOD] Invalid ammobox id provided (3).
L 03/21/2021 - 17:35:37: [AMXX] Displaying debug trace (plugin "wpnmod_spawn_editor.amxx", version "0.5")
L 03/21/2021 - 17:35:38: [AMXX] Run time error 10: native error (native "wpnmod_get_ammobox_info")
L 03/21/2021 - 17:35:38: [AMXX]    [0] wpnmod_spawn_editor.sma::fw_LoadSpawnPointsCfg (line 1074)
L 03/21/2021 - 17:35:38: [AMXX]    [1] wpnmod_spawn_editor.sma::fw_MenuHandler (line 621)
```

How i understood, this problem in module, not in plugin. 

Here is the fix for this issue: https://aghl.ru/forum/viewtopic.php?f=42&t=721&hilit=weaponmod+0.8&start=290
30th page, second comment, problem #2.

**Comment by tmp64** [comment]
Sorry for taking so long, didn't really have free time. AGHL is down at the moment - can't see the fix. Can you please post the plugin somewhere so I can debug it?

**Comment by tmp64** [comment]
@ST380011A submitted a fix in #2. Check if it resolved the issue.

**Comment by xRiseless** [comment]
> @ST380011A submitted a fix in #2. Check if it resolved the issue.

It works. Thanks
